gui/src/app/init/topology/relation.js

```diff
@@ -648,6 +650,7 @@ class RelationModule {
     // If we landed on a rect, add relation, otherwise, cancel.
     if (rect) {
       self.ambiguousAddRelationCheck(endpoint, self, rect);
+      this.analytics.sendEvent(self.analytics.ADD);
```

Comment:
should `this` be `self`? 

doesn't this need a label or a category?

Reply:
This will result in `Canvas > Relation` action: Add. Do you think it needs something else?

Reply:
Nope that will be fine